docs/markdown/Python/python/python-third-party-dependencies.md

```diff
@@ -479,28 +479,34 @@ Django @ file:///Users/pantsbuild/prebuilt_wheels/django-3.1.1-py3-none-any.whl
 
 There are two mechanisms for setting up custom Python distribution repositories:
 
-#### Simple repositories as defined by PEP 503
+#### PEP-503 compatible indexes
 
-If your custom repo is of this type, i.e., "private PyPI", aka "cheese shop", use the option `indexes` in the `[python-repos]` scope.
+Use `[python-repos].indexes` to add PEP 503-compatible indexes, like PyPI.
 
 ```toml pants.toml
 [python-repos]
 indexes.add = ["https://custom-cheeseshop.net/simple"]
 ```
 
-To exclusively use your custom index—i.e. to not use PyPI—use `indexes = [..]` instead of `indexes.add = [..]`.
+To exclusively use your custom index, i.e. to not use the default of PyPI, use `indexes = [..]`
+instead of `indexes.add = [..]`.
 
-#### A Pip findlinks repository
+#### pip `--find-links`
 
-If your custom repo is of this type, use the option `repos` in the `[python-repos]` scope.
+Use the option `[python-repos].find_links` for flat lists of packages. Same as pip's `--find-links`
+option, you can either use:
+
+* a URL to an HTML file with links to wheel and sdist files, or
+* a `file://` absolute path to a local directory with wheel and/or sdist files.
```

Comment:
I'm almost positive a bare absolute path works too. It does in Pex. Perhaps it complicates docs too much and should just be an easter egg though since it adds 0 capabilities.